test-framework/junit5-config/src/main/java/io/quarkus/test/config/LoggingSetupExtension.java

```diff
@@ -11,7 +13,12 @@
  * test), but also for getting instant log output from {@code QuarkusTestResourceLifecycleManagers} etc.
  */
 public class LoggingSetupExtension implements Extension {
+    private static final String JUNIT_QUARKUS_ENABLE_BASIC_LOGGING = "junit.quarkus.enable-basic-logging";
+
     public LoggingSetupExtension() {
-        LoggingSetupRecorder.handleFailedStart();
+        SmallRyeConfig config = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
+        if (config.getOptionalValue(JUNIT_QUARKUS_ENABLE_BASIC_LOGGING, boolean.class).orElse(true)) {
+            LoggingSetupRecorder.handleFailedStart();
+        }
```

Comment:
If we are completely sure the logging layer is initialized consistently with what is initialized with a normal start, maybe we should just avoid setting the handlers again when the app starts? I.e. do nothing when we set the handlers if they are already defined.

Now what I'm not sure of is how much the delayed handler is shared between tests.

Reply:
There is a single delayed handler, so I think that is not the issue.

We could try to avoid setting the handlers again. It may be tricky, because this is part of the regular Quarkus start, which means we have to create some different execution paths.